libraries/provision/ansible/playbooks/tasks/fetch-sync-gateway-all-logs.yml

```diff
@@ -0,0 +1,60 @@
+# Fetch sync gateway logs for CentOS
```

Comment:
I think this was added when fetch-sync-gateway-logs has old code. It was retrieving only 2 logs , but I need all logs and I do not want to edit existing log as it might effect exist tests which can mess . I got these new changes after I pulled changes from master and these new changes looks like change of getting all logs. I changed the test to use fetch-sync-gateway-logs and tested it . I will remove fetch-sync-gateway-all-logs.yml